commands/music/play.js

```diff
@@ -0,0 +1,75 @@
+/* eslint linebreak-style: 0 */
+const Command = require(`${process.cwd()}/base/Command.js`);
+const { MessageEmbed } = require("discord.js");
+const ytapi = require("simple-youtube-api"); 
```

Comment:
Could simply put this into the `../index.js` file, and call it through **this.client.youtube**. That way it follows with other apis we use.